ThemeConfig.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
```

Comment:
Were there any project file changes that arose in your `git diff` when you added this file?

Reply:
Kinda confused on what you mean. It was in the diff because it was a new file.

Reply:
XCode has some metadata that it tracks about files, not like other IDEs I've used. Files have to be "Added" to the project with build targets specified on each one. That info is kept in the project files, so it has to be committed as well.

Reply:
Lovely. I saw that file and didn't commit it. And then did a hard reset so lost the changes. I can just delete this file and commit it again and maybe that'll fix it

Reply:
You can go into XCode at File > Add File to make it show up.

Reply:
I like this color file, btw. Muchas gracias. Me encanta su trabajo.

Reply:
Gracias. Acabo de cometer el archivo que pediste 📝

Reply:
Do you see that there are two `ThemeConfig.swift` files now?
Eres un tonto.

Reply:
Cabron, que me importa. Ya voy a dormir. Lo haré mañana

Reply:
Bien. Puedes. Hasta mañana.

Reply:
Pienso que es bueno.